configuration/index.md

```diff
@@ -160,7 +160,7 @@ Please be aware that not every addon/binding of openHAB version 1.x has been upd
 You can still run these but you need to use file configuration for them.
 
 Things and Items can either be defined and managed in configuration files or handled by Paper UI in a [system-side database](/docs/administration/jsondb.html).
-For Things and Items, both methods can be used **side-by-side**, e.g. a Thing can be discovered and created by Paper UI and the Items that link to that very same Thing (that Thing's Channels, actually) can be manually defined in a `.items` configuration file. 
+You can use PaperUI for Things and files for Items **side-by-side**, e.g. a Thing can be discovered and created by Paper UI and the Items that link to that very same Thing (that Thing's Channels, actually) can be manually defined in a `.items` configuration file. But you **must not** use both methods in parallel for Items (and if you want to do yourself a favor, also not for Things).
```

Comment:
New sentences should go on a new line for better tracking in git. But I disagree with this addition, since there is no issue using a mix of both managed and unmanaged Items and Things at the same time, at least that I am aware of. This may be a good place to use tabs though, to separate Beginner and Advanced functionality, and not even discuss the possibility of unmanaged Items and Things on the Beginner tab. @Confectrician, Do you think this would be a good use of tabs, or do you have other plans for separating Beginner, Advanced, Developer/Expert functionality?

Reply:
I would disagree with this too, as a general information.

We could add a recommendation for beginners to choose one system fo rless confusion, but generally and even mor important technically it is fine to mix those methods.

I am doing this myself currently as a partly migration from files to json db based config.
Since the configuration part of the docs adresses all users in all expirience levels, we should be careful with DOs and DONTs.

I don't know if tabs are a good usecase here.
I would suggest a notice/warning here with a beginner recommendation.

Reply:
Well I made this change because of some beginner to misunderstand the previous text. 
But this is a good example of the quirks to come with developers writing manuals for users -
they take their developer perspective and think they even do users a favor because they coded things to allow for mixing so want to have this available to and known by users.
Yes technically you can do GUI+files for items (and for things) but even for skilled users like ourselves, that is a bad idea. They/we quickly forget about that mixing and can run into trouble later because of this so it clearly not advisable to _anyone_. So I feel my change to be the better default than what's there.
Do you have a better wording ?
I don't want to publish wrong information, but anyone mixing (not just beginners but including pros!) should at least be made aware of the risk twice.

Reply:
> Yes technically you can do GUI+files for items (and for things) but even for skilled users like ourselves, that is a bad idea. 

I have a mixed environment and have not had any issues. What problems are you trying to avoid by recommending that users not mix managed and unmanaged items/things? As Jerome mentioned, a warning containing these issues might suffice. Maybe recommend managed and add a side note for advanced users?

BTW, one thing that is confusing to me with the current doc is the use of the phrase "side-by-side"... this should be removed or reworded.